CMakeLists.txt

```
@@ -7,7 +7,8 @@ set(PROJECT_VERSION_MINOR 5)
 set(PROJECT_VERSION_PATCH 1)
 set(PROJECT_VERSION "${PROJECT_VERSION_MAJOR}.${PROJECT_VERSION_MINOR}.${PROJECT_VERSION_PATCH}")
 
-option(BUILD_STATIC "Build static version of libunshield" OFF)
+option(LIBUNSHIELD_BUILD_STATIC "Build static version of libunshield" OFF)
+option(LIBUNSHIELD_BUILD_TESTS "Build test of libunshield for CTest" OFF)
```

```suggestion
option(BUILD_TESTING "Build libunshield tests for CTest" OFF)
```
No need to introduce a second option, *BUILD_TESTING* is already available if using CTest, see https://cmake.org/cmake/help/latest/module/CTest.html for more information.